runtime/ops/os/mod.rs

```diff
@@ -341,6 +343,120 @@ fn op_uid(
   Ok(None)
 }
 
+#[derive(Serialize)]
+struct CpuUsage {
+  system: usize,
+  user: usize,
+}
+
+#[op2]
+#[serde]
+fn op_runtime_cpu_usage() -> CpuUsage {
+  let (sys, user) = get_usage();
+  CpuUsage {
+    system: sys.as_micros() as usize,
+    user: user.as_micros() as usize,
+  }
+}
+
+#[cfg(unix)]
+fn get_usage() -> (std::time::Duration, std::time::Duration) {
```

Comment:
Maybe call these `get_cpu_usage` as there are also memory usage functions in this file.